ant-node/src/put_validation.rs

```diff
@@ -614,24 +614,30 @@ impl Node {
         // verify quote timestamp
         let quote_timestamp = payment.quote.timestamp;
         let quote_expiration_time = quote_timestamp + Duration::from_secs(QUOTE_EXPIRATION_SECS);
-        let quote_expiration_time_in_secs = quote_expiration_time
+        let _quote_expiration_time_in_secs = quote_expiration_time
             .duration_since(UNIX_EPOCH)
             .map_err(|e| {
                 Error::InvalidRequest(format!(
                     "Payment quote timestamp is invalid for record {pretty_key}: {e}"
                 ))
             })?
             .as_secs();
+        // NB TODO @mick: can we check if the quote has expired with block time in evmlib? Or should nodes do it manually here? Else keep the block below
```

Comment:
the `quote` was made by the node, not related to the block time, hence have to do it manually ?
unless node using block time on quote, which requires an extra query to the block?

Reply:
We can't really couple a transaction to a quote anymore since we no longer use the transaction's event logs to verify payments. So we'd have to do it manually.